test/unit/spec/EnrollTotpController_spec.js

```diff
@@ -697,6 +700,12 @@ function (_, $, Q, OktaAuth, LoginUtil, StringUtil, Util, DeviceTypeForm, Barcod
         });
         itp('has correct fields displayed when different dropdown options selected', function () {
           return enrollOktaPushGoCannotScan()
+          .then(function (test) {
+            return test.manualSetupForm.waitForDropdownElement(test);
+          })
+          .then(function (test) {
+            return test.manualSetupForm.waitForCountryCodeSelect(test);
```

Comment:
do we need both ```waitFor``` calls? I think if the page has a form with a dropdown and a country select, if we wait for one it should be enough.

Reply:
We should wait for the Chosen plugin being initialized on both elements.
Sometimes in a browser, it can get not in time.
Have to be sure, that Chosen is ready for both dropdowns.